### PR TITLE
remove fixed footer also

### PR DESCRIPTION
**Pull request by DrMeers**

### CODE REVIEW

File: members/W000790.yaml [hunk 1]
```diff
@@ -7,7 +7,8 @@ contact_form:
     - wait:
         - value: 3
     - javascript:
-        - value: "document.querySelector('.elementor-location-header').remove();"
+        - value: "document.querySelector('.elementor-location-header')?.remove();"
+        - value: "document.querySelector('.elementor-location-footer')?.remove();"
```
Comment (j-ro) [inline members/W000790.yaml:11]: does this work on your implementation? It doesn't on ours, you need separate javascript statements.

**Review by j-ro — CHANGES_REQUESTED** [review]

File: members/W000790.yaml (hunk 1 above)
Comment (DrMeers) [reply members/W000790.yaml:11]: yes this works fine on our implementation, but I can separate the statements for yours

Comment (DrMeers) [reply members/W000790.yaml:11]: yours would also have issues with https://github.com/unitedstates/contact-congress/commits/master/members/S000148.yaml etc then?

Comment (DrMeers) [reply members/W000790.yaml:12]: see commit below please @j-ro

**Pull request merged**